lib/tasks/plugin_test.rake

```diff
@@ -0,0 +1,99 @@
+require File.expand_path('../../config/environment', File.dirname(__FILE__))
+
+desc 'Run a single or multiple plugins tests - called via rake plugin:test[plugin_name,plugin_name]'
+task 'plugin:test' => [:environment] do |t, args|
+  plugin_names = args.extras
+    
```

Comment:
There are a few instances of whitespace in this file, please just clean that up.